libminifi/include/utils/Void_t.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace utils {
+
+// ppartial detection idiom impl
```

Comment:
`void_t` is used by, but not really part of the detection idiom. Could you move the comment line back to StringUtils.h?

I forgot to add a reference to cppreference.com in the original. When moving it back, could you please append `, from cppreference.com` to the comment and fix the typo (`s/ppartial/partial/`)?

So my ask summarized:
- remove this comment line
- Add the line below to where it was removed from:
```
// partial detection idiom impl, from cppreference.com
```

Reply:
Changed as requested.

Reply:
the comment belongs to stringutils.h